ocs_ci/utility/utils.py

```diff
@@ -366,25 +373,33 @@ def run_cmd(cmd, **kwargs):
         (str) Decoded stdout of command
 
     """
-    log.info(f"Executing command: {cmd}")
-    if isinstance(cmd, str):
-        cmd = shlex.split(cmd)
-    r = subprocess.run(
-        cmd,
-        stdout=subprocess.PIPE,
-        stderr=subprocess.PIPE,
-        stdin=subprocess.PIPE,
-        **kwargs
-    )
-    log.debug(f"Command output: {r.stdout.decode()}")
-    if r.stderr and not r.returncode:
-        log.warning(f"Command warning:: {r.stderr.decode()}")
-    if r.returncode:
-        raise CommandFailed(
-            f"Error during execution of command: {cmd}."
-            f"\nError is {r.stderr.decode()}"
+    try:
+        quiet_state = quiet_logs(quiet=kwargs.pop('quiet', False))
+        log.info(f"Executing command: {cmd}")
+        if isinstance(cmd, str):
+            cmd = shlex.split(cmd)
+        r = subprocess.run(
+            cmd,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.PIPE,
+            stdin=subprocess.PIPE,
+            **kwargs
         )
-    return r.stdout.decode()
+        log.debug(f"Command output: {r.stdout.decode()}")
+        if r.stderr and not r.returncode:
+            log.warning(f"Command warning:: {r.stderr.decode()}")
+        logging.disable(logging.NOTSET)
```

Comment:
2. 'Try:' clause should end here

Reply:
Any reason not to end it directly after `subprocess.run`? 
I don't think `if r.stderr and not r.returncode...` can cause an exception

Reply:
ok